src/test/java/org/sonar/plugins/findbugs/FindbugsExecutorTest.java

```diff
@@ -20,6 +20,7 @@
 package org.sonar.plugins.findbugs;
 
 import com.google.common.collect.Lists;
+import com.sun.org.apache.xpath.internal.operations.Bool;
```

Comment:
seems needless dependency on internal and repackaged class

Reply:
Yes this was a mistake but I didn't notice until I opened the PR (and thanks for heads up - I didn't realize this was an internal dependency)